### DISCUSSION TITLE
Welcome to AspNetCore-WebApi-Course Discussions!

### DISCUSSION THREAD

**Discussion by mjebrahimi** [dotnetzoom/AspNetCore-WebApi-Course#152] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by javaddelfani-asp** [comment]
سلام و عرض ادب خدمت شما و دوره جامع و عالیتون
ببخشید برای اینکه بتونیم استتوس کد 204 رو در قالب کلاس | ApiResult نمایش دهیم چکار باید کرد؟
من یک کلاس مثل BadRequestException ایجاد کردم ولی در قالب کلاس دلخواه نمایش داده نمیشه
لطفا راهنمایی کنید
